### PR TITLE
DEVELOPER-2782 - Developer blog dcp logic fixed to include updated urls.

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

**Comment by LightGuard** [comment]
@lincolnthree will you please verify, you initially reported the issue.

**Comment by LightGuard** [comment]
@lincolnthree if you look at the checks there's a site preview check, you can click on the "Details" link and it'll go to the site preview.

File: javascripts/buzz.js [hunk 1]
```diff
@@ -119,11 +119,12 @@ app.buzz = {
...
             }
             d.updatedDate = jQuery.timeago(new Date(d.sys_created));
+            var dateOfBuzz = moment(d.sys_created).format('YYYY/DD/MM')
             d.sys_description = d.sys_description[0].substr(0,197) + '...';
-            if (d.sys_url_view[0].startsWith('http://developerblog.redhat.com')) {
+            if (d.sys_url_view[0].startsWith('http://developerblog.redhat.com') || d.sys_url_view[0].startsWith('https://developerblog.redhat.com') || d.sys_url_view[0].startsWith('http://developers.redhat.com/blog/') || d.sys_url_view[0].startsWith('https://developers.redhat.com/blog/')) {
               d.permanentLink = d.sys_url_view;
             } else {
-              d.permanentLink = "//planet.jboss.org/post/" + d.sys_content_id;
+              d.permanentLink = "//developers.redhat.com/blog/" + dateOfBuzz +"/" + d.sys_content_id;
```
Comment (paulrobinson) [inline javascripts/buzz.js]: I'm not sure this 'dateOfBuzz' does what you expect. See the buzz page errors on: http://developers-pr.stage.redhat.com//pr/963/blinkr/671/report.html

**Comment by Dantheman720** [comment]
retest this please

**Comment by paulrobinson** [comment]
@Dantheman720 can you drive the review process please. You need a code review, and a review from the requestor.

**Comment by lincolnthree** [comment]
Doesn't appear that this is working? I still see links from developerblog.redhat.com and planet.jboss.org

**Comment by lincolnthree** [comment]
Also, best to find me on hipchat when you need a review. Github notifications are not a super reliable way to get a hold of me (lots of noise)..

**Comment by LightGuard** [comment]
I'm seeing this change working, I'm going to merge

**Pull request merged**